README.md

```diff
@@ -221,6 +221,20 @@ tensorflowjs_converter \
 
 The time of first call also includes the compilation time of WebGL shader programs for the model. After the first call the shader programs are cached, which makes the subsequent calls much faster. You can warm up the cache by calling the predict method with an all zero inputs, right after the completion of the model loading.
 
+6. I have models converted with previous version of TensorFlow.js converter (0.15.x), they contain model file in .pb extension, how do I convert them into new json output format?
```

Comment:
`I have a model converted with a previous version of the TensorFlow.js converter (0.15.x) that is in .pb format. How do I convert them to the new JSON format?`